src/js/providers/html5.js

```diff
@@ -482,9 +482,6 @@ define([
                 // Playback rate is broken on Android HLS
                 _this.supportsPlaybackRate = false;
             }
-            if (source.preload && source.preload !== _videotag.getAttribute('preload')) {
-                _setAttribute('preload', source.preload);
-            }
```

Comment:
That depends on when the source is set relative to when preload it. This is also one of the reasons why preload is an item/source property - so that they can be set at the same time or correct order, but now the player needs be in control of preload so the source's setting is completely irrelevant (so why are we still setting preload on source?).

Setting preload on the video to none here might be the best solution. It the only way to prevent any preloading until the player explicitly calls preload with the desired setting. This also means we do not need to call provider.preload() with a value of "none" since it would do nothing.